### DISCUSSION TITLE
Linux version

### DISCUSSION THREAD

**Discussion by lcarcamo1526** [SauceLLC/sauce4zwift#10] [Ideas]
Hey @mayfield great work!, I've been using Sauce for Strava and I loved it!, I tried one of the releases of sauce4zwift previously on Windows and worked like a charm, I was just wondering which steps are necessary to run it on Linux, I run Zwift on Lutris and pair the sensors through the companion app. 
Btw i use arch ;)

**Comment by mayfield** [comment]
Can you try this build?  If this works I can cheaply integrate it into my build "pipeline" (i.e. the steps I run every release)

https://github.com/SauceLLC/sauce4zwift-releases/releases/download/v0.10.7-alpha.3/sauce4zwift-0.10.7-alpha.3.AppImage

**Reply by lcarcamo1526** [reply]
After some testing in 2 different distros with different setups, here is my experience:

```
Kernel: x86_64 Linux 5.19.13-arch1-1
Shell: zsh 5.9
Resolution: 3840x1080
WM: i3 (Xorg)/  GNOME (Xwayland) 
GTK Theme: Adwaita [GTK3]
CPU: AMD Ryzen 5 2600 Six-Core @ 12x 3.4GHz
GPU: NVIDIA GeForce GTX 1060 3GB
RAM: 15942MiB
```

Every single feature related to Zwift streaming was working flawless, events, chat, watts, speed, etc.  (Even auto-update, which is great!!)
But the funny thing happened when running Zwift with 2 monitors.
By default, the panels' init in a fix location, in a just one of the screen and they get fixed, for some reason I could not move it or resize the panels, some of them for some reason could not be closed, so I think it is completely related to gnome I tried using Xorg and got the exactly same behavior.
What I was doing to solve this little issue was open sauce4zwift first and then connect my second monitor, so I think it would be great to choose which screen would be the default one.

Thank you again for your patience and this fascinating tool!
